src/main/webapp/index.html

```diff
@@ -33,7 +33,13 @@ <h1>PIECED</h1>
             With Pieced, we aimed to bring the concept of visualizing animal species through their population to a larger scale, providing additional information about the species along the way.
             <br>
             <br>
-            <small class="text-muted">Created by Fiza Goyal, Ariana Zhu, and Maxwell Chen for Google's 2020 STEP internship program.</small>
+            <small class="text-muted">Created by Fiza Goyal, Ariana Zhu, and Maxwell Chen for the Google STEP internship program in Summer 2020. They were hosted by Tom Russ and Dave Newman.</small>
+            <br>
+            <small class="text-muted">
+                <a href="https://github.com/fgoyal"> Fiza</a> is a rising junior ('22) at University of Illinois at Urbana-Champaign.
+                <a href="https://github.com/arianazhu"> Ariana<a> is a rising junior ('22) at Purdue University.
+                <a href="https://github.com/maxhchen"> Maxwell</a> is a rising junior ('22) at UC Berkeley.
```

Comment:
I think maybe it'd be nice to have a little more info kind of explaining the STEP program? Can you also write something about how it's open source and add a link to the github for the project?

Reply:
What kind of info were you thinking of for the STEP program? Something like "12-week internship for 1st and 2nd year undergraduate students"?

Yup, will add that it's open-source and add a GitHub link

Reply:
Yeah I think something like that sounds good? I was just picturing more of a paragraph like "This webapp was created during the Google STEP Internship in summer 2020. STEP is a 12 week program for 1st and 2nd year undergraduate students. etc"